src/modules/mock.command.js

```diff
@@ -0,0 +1,12 @@
+const command = require('./command');
+
+const mock = async function(response, { input, i18n }) {
+	const {user} = response 
+	var lastWord = input.substring(input.lastIndexOf(" ")+1);
```

Comment:
- dont use var
- need separate constructions and put into substring calculated variable